### PR TITLE
allow for functions to run before main loop

### PR DESCRIPTION
**Pull request by kwstannard**
Basically this pull request but without any refactoring: https://github.com/gocardless/hutch/pull/206

### CODE REVIEW

File: lib/hutch/worker.rb [hunk 1]
```diff
@@ -20,6 +21,7 @@ def initialize(broker, consumers)
     # never returns.
     def run
       setup_queues
+      setup_procs.each(&:call)
```
Comment (michaelklishin) [inline lib/hutch/worker.rb]: Why does the worker invoke those callables? Is it to make sure the queues are set up first?

Comment (kwstannard) [inline lib/hutch/worker.rb]: Because the worker sets up the queues, I figured that the other setup stuff should go here as well.

**Comment by kwstannard** [comment]
@michaelklishin if I were to move the queue setup back into its own class for the setup loop, would you like me to move the setup loop out into the CLI?

**Comment by michaelklishin** [comment]
@kwstannard I don't think we need to move queue setup to a separate class.

**Pull request merged**